app/model/LatestAnalyticsBreakdownItem.scala

```diff
@@ -0,0 +1,35 @@
+package model
+
+import ai.x.play.json.Jsonx
+import play.api.libs.json.Format
+
+case class LatestAnalyticsBreakdownItem(uniques: Long, pageviews: Long, timeSpentOnPage: Option[Double] = None)
+
+object LatestAnalyticsBreakdownItem {
+
+  import cats.Semigroup
+  import cats._
+  import cats.implicits._
+
+  implicit val latestCampaignAnalyticsBreakdownItemFormat: Format[LatestAnalyticsBreakdownItem] =
+    Jsonx.formatCaseClass[LatestAnalyticsBreakdownItem]
+
+  implicit val latestAnalyticsMonoid: Monoid[LatestAnalyticsBreakdownItem] =
+    new Monoid[LatestAnalyticsBreakdownItem] {
+      override def combine(itemOne: LatestAnalyticsBreakdownItem,
+                           itemTwo: LatestAnalyticsBreakdownItem): LatestAnalyticsBreakdownItem = {
+        LatestAnalyticsBreakdownItem(
+          uniques = itemOne.uniques + itemTwo.uniques,
```

Comment:
What context is this addition done in? Uniques usually can't be added together unless it's over the same range + more days?

Reply:
We discussed and it's safe we believe in this context. But context is important.